testing/tests/DevExpress.ui.widgets/fileManagerParts/navigation.tests.js

```diff
@@ -151,4 +152,41 @@ QUnit.module("Navigation operations", moduleConfig, () => {
         assert.equal(target, "TestRFN", "Custom is correct");
     });
 
+    test("splitter should change width of dirs tree and file items areas", function(assert) {
+        renderer.fn.width = function() {
```

Comment:
You need to restore the original function `width()` at the end of the test, otherwise other tests will use this modified function and they will fail because of this.